references.md

```diff
@@ -0,0 +1,28 @@
+# References
+
+## Tools
+
+| Topic | Link |
+| -- | -- |
+| PHP for beginners | http://www.sololearn.com/Play/PHP# |
+| Git - try git | https://try.github.io/levels/1/challenges/1 |
```

Comment:
Topic can be have the link for example `[PHP for beginners](http://www.sololearn.com/Play/PHP)`